adapter/src/main/java/com/twilio/kudu/dataloader/generator/ColumnValueGenerator.java

```diff
@@ -24,4 +24,9 @@
 
   public abstract Object getColumnValue(String columnName);
 
+  // needed because while reading from json the default constructor is used
+  // without member
+  // variables being initialized first
+  public abstract void initialize();
```

Comment:
Probably should be Javadoc formatted
```suggestion
  /** 
    * Initialize the {@code ColumnValueGenerator} from JSON or other files. Used as a alternative constructor
    */
  public abstract void initialize();
```